calendar/experiments/calendar/parent/ext-calendar-calendars.js

```diff
@@ -18,15 +18,22 @@ this.calendar_calendars = class extends ExtensionAPI {
       getResolvedCalendarById,
       isOwnCalendar,
       convertCalendar,
-    } = ChromeUtils.import("resource://ext-calendar-draft/api/ext-calendar-utils.jsm");
+    } = ChromeUtils.import(this.extension.rootURI.resolve("experiments/calendar/ext-calendar-utils.jsm"));
```

Comment:
This path is of course always dependent on the developer's own structure, but since this experiment also has an add-on with background scrip I think we should use this:

```suggestion
    } = ChromeUtils.import(this.extension.rootURI.resolve("api/ext-calendar-utils.jsm"));
```

Reply:
I fixed the add-on in the same commit, and tested it in both 78 and 83 (see the renames and changes to the manifest?).

I would like to get the draft into a state where the developer including it into their add-on does not need to touch the implementation at all, thus we should use a folder name that contains 'calendar' or something like that. I personally like `experiments/calendar`, as that permits add-on authors to cleanly add additional experiments if necessary. But of course it would also be possible to name it `api/calendar` or `api-calendar` or something else – but just `api` is a bat fit imho.

Reply:
Thats fair. Let's keep experiments/calendar then.